internal/broker/config.go

```diff
@@ -53,6 +94,38 @@ func getDropInFiles(cfgPath string) ([]any, error) {
 	return dropInFiles, nil
 }
 
+func parseUsersSection(cfg *userConfig, users *ini.Section) {
+	if users == nil {
+		// The default behavior is to allow only the owner
+		cfg.allowedUsers = make(map[string]bool)
+		cfg.allowedUsers[OwnerUserKey] = true
+		return
+	}
+
+	cfg.homeBaseDir = users.Key(homeDirKey).String()
+	cfg.allowedSSHSuffixes = strings.Split(users.Key(sshSuffixesKey).String(), ",")
+	// We need to differentiate unset owner from empty owner.
+	// - Unset means that the owner will be autoregistered
+	// - Empty means that there is no owner
+	if users.HasKey(ownerKey) {
+		o := users.Key(ownerKey).String()
+		cfg.owner = &o
+	}
```

Comment:
this feels a bit counterintuitive as we checked already and put OWNER, wondering if we should have a flag instead saying "no owner so first registration needed" and leave the list empty?

just throwing it out there